spec/fixtures/responses/whois.aero/status_reserved.expected

```diff
@@ -0,0 +1,50 @@
+#disclaimer
+  %s == nil
+
+
+#domain
+  %s == nil
+
+#domain_id
+  %s == nil
+
+
+#status
+  %s == :reserved
+
+#reserved?
+  %s == true
+
+#available?
```

Comment:
Can you also add `registered?`. Please use the following order

```
#available?
  %s == false

#registered?
  %s == true

#reserved?
  %s == true
```

Technically, `reserved` is not a standard property yet (therefore I generally added it to the bottom of the expectation file), but it may probably make sense here.